### PR TITLE
Add "contributing" to Readme

### PR DESCRIPTION
**Pull request by maphew**
Added some helpful navigation links to the wiki pages in main project that I've found myself needing to reference frequently while learning how to add new programs to the Extras bucket.

Something that's missing but I don't quite know how to describe is:  what's the workflow is. E.g. 
1. fork
2. create branch (what naming convention? `appname-patch-1`?)
3. write app.json
4. test locally (which guide page(s)?)
5. PR
?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,12 @@ Extra manifests for [Scoop](https://scoop.sh), the Windows command-line installe
 To make it easier to install apps from this bucket, run
     `scoop bucket add extras`
 
-## contributing
-For documentation on how to add packages to the Extras bucket see [the Wiki](https://github.com/ScoopInstaller/Scoop/wiki) from main [Scoop](https://github.com/ScoopInstaller/Scoop) project, particularly: 
+## How do I install these manifests?
+
+To add this bucket, run `scoop bucket add extras`. To install, do `scoop install <manifest>`.
```
Comment (rasa) [inline README.md:10]: This repeats the same instruction found on lines 5-6, but your verbiage is better. I would love to see us have standard verbiage across all the buckets we maintain. Perhaps the verbiage could live in https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md as well.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,12 @@ Extra manifests for [Scoop](https://scoop.sh), the Windows command-line installe
...
+
+## Contributing
+For documentation on how to add packages to the Extras bucket see [the Wiki](https://github.com/ScoopInstaller/Scoop/wiki) from main [Scoop](https://github.com/ScoopInstaller/Scoop) project, particularly:
```
Comment (rasa) [inline README.md:13]: Using the word `main` here could be slightly confusing as we have a repo/bucket named `main` as well.

**Comment by rashil2000** [comment]
Check the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

**Comment by maphew** [comment]
I'm glad part of this was seen as a useful improvement, thanks for using it! 

The merge update by @rashil2000 to _Contributing.md_ skips over the challenge I faced (but didn't clearly explain). Namely: within the repository Extras itself there is little guidance on what to do to contribute. I have only cloned this repo as I'm not doing anything with the others, and certainly not Scoop itself. That's beyond my pay grade! On a related note, up until this thread I had never seen _Contributing.md_. I don't believe it's mentioned in the wiki, which I've been using as my prinary learning guide.

So maybe what's needed for each of the Bucket readme's is simply a link to the wiki? and from the wiki to Contributng.md? (or perhaps move Contributing to the wiki so all docs are in one place.)

**Comment by rashil2000** [comment]
The Contributing Guide link is shown whenever someone creates an issue or a PR.

But you're right, it should be present in the Wiki too. I'll add it.

**Comment by maphew** [comment]
Oh. So it is. Not seeing it was a result of my own act then. Sorry for that bit of noise. I succumbed to a form of banner blindness ("I've seen Contibuting on a hundred other github projects, I've read it already.") Still a good idea IMO to link from  Bucket readme's to Wiki.

**Comment by rashil2000** [comment]
Done.

**Pull request closed without merging**